README.md

```diff
@@ -1,75 +1,74 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Movie Capstone Project
 
-> Description the project.
+> A website display list of movies from an external API. The webapp consist of tow user interfaces. It have home page with movies and popup window with detail of given movie. User can also add comments and view comments on popup window.
 
+![](./src/assets/images/movie.png)
+![](./src/assets/images/movie0.png)
+![](./src/assets/images/movie1.png)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html
+- CSS
+- Javascript
+- Movie API
+- Webpack
+- JEST
+- ES6
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
+git clone https://github.com/letarulinda/movie-capstone
 
 ### Prerequisites
 
+- [Movie API](https://api.tvmaze.com/shows)
+- `Webpack installation`
+- `SSH key for github linked to git on local environment`
+- `Html & CSS basics`
+
 ### Setup
 
+- If it suits you, on your local computer, create a directory in your preferred location (e.g in the Desktop or Document directory)
+
 ### Install
 
 ### Usage
 
 ### Run tests
 
-### Deployment
-
+- Install project dependencies (libraries) using `npm install` command
+- Run webpack build using npm run build command
+- Run webpack dev server using npm run start command
 
+### Deployment
 
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+### Video of My project
+file:///C:/Users/BRIAN/Documents/Zoom/2022-10-29%2016.33.31%20Linda%20Masia's%20Zoom%20Meeting/video2232221055.mp4
```

Comment:
- [ ] As the previous reviewer mentioned, This is the local link of the recorded zoom meeting and I can't review it. Please upload this video to google drive and provide the link in the pull request summary or here in the README file, or Please use.